lib/vsc/mympirun/rm/sched.py

```diff
@@ -106,11 +106,15 @@ def __init__(self, options=None, **kwargs):
         self.nodes = None
         self.set_nodes()
 
-        self.ppn = None
-        self.set_ppn()
+        self.multiplier = None
+        self.set_multiplier()
 
-        self.mpiprocesspernode = None
-        self.set_mpiprocesspernode()
+        ppn = os.environ.get('PBS_NUM_PPN')
+        if ppn is not None:
+            self.ppn = int(ppn)
+        else:
+            LOGGER.error('PPN is None')
```

Comment:
this will just print an error, not raise an exception

also, mention `$PBS_NUM_PPN`: `Failed to determine # cores per node (ppn) via $PBS_NUM_PPN`